samples/client/echo_api/ruby-httpx/lib/openapi_client/models/number_properties_only.rb

```diff
@@ -83,14 +83,16 @@ def initialize(attributes = {})
     def list_invalid_properties
       warn '[DEPRECATED] the `list_invalid_properties` method is obsolete'
       invalid_properties = Array.new
-      if !@double.nil? && @double > 50.2
-        invalid_properties.push('invalid value for "double", must be smaller than or equal to 50.2.')
-      end
+      if !@double.nil?
+        if !@double.nil? && @double > 50.2
```

Comment:
both 86 and 86 do a nil check. is that expected?

Reply:
I don't know. Maybe? I don't know Mustache templating at all, so trying to fix this has already been much more of a time suck than I anticipated and we already fixed this issue to our own satisfaction in our own deployment, so we don't need this. I don't think I'm going to contribute on this PR any further, feel free to do whatever you like with it.